data/metadataSchema.json

```
@@ -0,0 +1,163 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Field Metadata for CSAF Documents",
+  "description": "JSON schema for json file storing metadata on fields of CSAF documents",
+  "type": "object",
+  "$defs": {
+    "relevance_levels_t": {
+      "title": "Relevance Levels",
+      "description": "Defines the possible relevance levels",
+      "type": "string",
+      "enum": [
+        "mandatory",
+        "best_practice",
+        "want_to_have",
+        "nice_to_know",
+        "optional",
+        "excluded"
+      ]
+    },
+    "documentation_file_path_t": {
+      "title": "File Path",
+      "description": "Defines the path to a file on the file system",
+      "type": "string",
+      "pattern": "docs/user(/.+)+-(spec|usage)(-csaf_(base|security_incident_response|informational_advisory|security_advisory|vex))?\\.[a-z]{2}\\.md"
```

BCP 47 is a complex area...
```suggestion
      "pattern": "^docs/user(/.+)+-(spec|usage)(-csaf_(base|security_incident_response|informational_advisory|security_advisory|vex))?\\.[a-z]{2,3}\\.md$"

---

Maybe also add a `minLength`?